test/lib.bash

```diff
@@ -199,19 +199,19 @@ function run_rolling_upgrade_tests {
 $base{{- with .Name }}
 {{- if eq .      "wathola-kafka-sender"}}{{.}}:v$eventing_kafka_image_version
 {{- else if eq . "kafka-consumer"      }}knative-eventing-kafka-broker-test-kafka-consumer:knative-v$eventing_kafka_broker_image_version
-{{- else if eq . "event-flaker"        }}{{.}}:v$eventing_image_version
-{{- else if eq . "event-library"       }}{{.}}:v$eventing_image_version
-{{- else if eq . "event-sender"        }}{{.}}:v$eventing_image_version
-{{- else if eq . "eventshub"           }}{{.}}:v$eventing_image_version
-{{- else if eq . "heartbeats"          }}{{.}}:v$eventing_image_version
-{{- else if eq . "performance"         }}{{.}}:v$eventing_image_version
-{{- else if eq . "print"               }}{{.}}:v$eventing_image_version
-{{- else if eq . "recordevents"        }}{{.}}:v$eventing_image_version
-{{- else if eq . "request-sender"      }}{{.}}:v$eventing_image_version
-{{- else if eq . "wathola-fetcher"     }}{{.}}:v$eventing_image_version
-{{- else if eq . "wathola-forwarder"   }}{{.}}:v$eventing_image_version
-{{- else if eq . "wathola-receiver"    }}{{.}}:v$eventing_image_version
-{{- else if eq . "wathola-sender"      }}{{.}}:v$eventing_image_version
+{{- else if eq . "event-flaker"        }}knative-eventing-test-{{.}}:knative-v$eventing_image_version
```

Comment:
so, this was broken?

Reply:
No, this one https://github.com/openshift-knative/serverless-operator/pull/1711/commits/776811291ce4461b0995e8aacaaa42871908f4dc